apps/src/code-studio/pd/application_dashboard/detail_view.jsx

```diff
@@ -7,9 +7,11 @@
  */
 
 import React, {PropTypes} from 'react';
-import {Button, FormControl, Table} from 'react-bootstrap';
+import {Button, FormControl} from 'react-bootstrap';
 import {Facilitator1819Program} from './detail_view_facilitator_specific_components';
-import _ from 'lodash';
+import Spinner from '../workshop_dashboard/components/spinner';
```

Comment:
I moved the spinner code to live in `pd/components` since we use it in multiple dashboards now. (that change is in staging)